agent/engine/docker_client.go

```diff
@@ -74,6 +77,8 @@ const (
 	inspectContainerTimeout = 30 * time.Second
 	removeImageTimeout      = 3 * time.Minute
 
+	ListPluginsTimeout = 1 * time.Minute
```

Comment:
This is arbitrary value, where did we get other timeout values from? 
One thing to note is that this will be used during agent startup to detect installed plugins as its capability, and I don't know if there's any implication of how long we can take to timeout during the startup.